sql/item_mods.sql

```diff
@@ -24570,8 +24574,544 @@ INSERT INTO `item_mods` VALUES (21460,5,5);
 INSERT INTO `item_mods` VALUES (21479,24,15);
 INSERT INTO `item_mods` VALUES (21479,26,5);
 INSERT INTO `item_mods` VALUES (21479,305,15);
+INSERT INTO `item_mods` VALUES (21506,8,10);
+INSERT INTO `item_mods` VALUES (21506,10,10);
+INSERT INTO `item_mods` VALUES (21506,25,25);
+INSERT INTO `item_mods` VALUES (21506,291,15);
+INSERT INTO `item_mods` VALUES (21507,8,15);
+INSERT INTO `item_mods` VALUES (21507,10,15);
+INSERT INTO `item_mods` VALUES (21507,25,30);
+INSERT INTO `item_mods` VALUES (21507,291,18);
+INSERT INTO `item_mods` VALUES (21606,28,14);
+INSERT INTO `item_mods` VALUES (21606,311,108);
+INSERT INTO `item_mods` VALUES (21606,432,30);
+INSERT INTO `item_mods` VALUES (21607,28,14);
+INSERT INTO `item_mods` VALUES (21607,311,108);
```

Comment:
When you add mods, please help reviewers by commenting what the mods you are adding are. It saves the hassle of having to open modifier.h and look them up to be sure everything is legit retail and that nobody mistakenly added skill that should have been item level skill. Even if they get wiped in a redump later, it helps reviewers.